examples/cache/front-envoy.yaml

```diff
@@ -31,7 +31,7 @@ static_resources:
             typed_config:
               "@type": "type.googleapis.com/envoy.extensions.filters.http.cache.v3alpha.CacheConfig"
               typed_config:
-                "@type": "type.googleapis.com/envoy.source.extensions.filters.http.cache.SimpleHttpCacheConfig"
+                "@type": "type.googleapis.com/envoy.extensions.cache.simple_http_cache.v3alpha.SimpleHttpCacheConfig"
```

Comment:
My suggestion would be to simply delete this example file rather than backport the API change.

Reply:
let me try and figure out why its happening - im pretty not keen on this approach either 

im also reluctant to delete the example as it matches to the related docs

Reply:
yep - this isnt going to work easily - ill use this pr to debug failing test...